herms.cabal

```diff
@@ -89,6 +89,7 @@ library
                      , split >=0.2 && <0.3
                      , text >= 1.1 && <1.3
                      , yaml >=0.08 && <0.11
+                     , regex-tdfa >=0.95
```

Comment:
What versions of `regex-tdfa` is this known to work with? 0.95 is a pretty old version. Is it also compatible with the 1.2.* versions? 

This should probably include an upper bound, see
 - https://pvp.haskell.org/ 
 - http://hackage.haskell.org/package/check-pvp

Reply:
I didn't get check-pvp to run but updated the lower and upper bounds.
Upper bound is now `<1.3` to prevent breaking changes and lower bound is `1.2.3`, as this looks to be the lowest version that runs with the required base. At least this is what I get from
```
rejecting: regex-tdfa-1.2.2 (conflict: base==4.11.1.0/installed-4.1...,
regex-tdfa => base>=4 && <4.11)
```
Don't know why `>=0.95` ended up in the code.